### PR TITLE
Code review conjiggerations

### PR DESCRIPTION
**Pull request by Snoldinator**

### CODE REVIEW

File: SSH TunnelBuilder/Models/ViewController.swift [hunk 1]
```diff
@@ -43,6 +47,20 @@ class ViewController: NSViewController, NSComboBoxDataSource {
         self.view.window?.title = "SSH TunnelBuilder"
     }
 
+    @objc private func refresh() {
+        viewModel.refresh { error in
+            if let error = error {
+                let alert = NSAlert()
```
Comment (Chaooz) [inline SSH TunnelBuilder/Models/ViewController.swift]: Alert box her burde være i en egen funksjon. Kan lage en statisk klasse for dette.

class MyUtils
{
  static func ShowAlertBox( message: String, alertStyle: NSAlert.Style )
  {
              DispatchQueue.main.async {
                    let alert = NSAlert()
                    alert.alertStyle = alertStyle            
                    alert.messageText = message
                    alert.addButton(withTitle: "OK")
                    alert.runModal()
                    return
                }
  }
}

File: SSH TunnelBuilder/Models/ViewController.swift [hunk 2]
```diff
@@ -107,6 +125,24 @@ class ViewController: NSViewController, NSComboBoxDataSource {
         tableView.reloadData()
         
     }
+    
+    func checkIcloudAccountStatus() {
+        
+        CKContainer.default().accountStatus { accountStatus, error in
+            if accountStatus == .noAccount {
+                DispatchQueue.main.async {
```
Comment (Chaooz) [inline SSH TunnelBuilder/Models/ViewController.swift]: Bytt ut med util klasse med ShowAlertBox
self.ShowAlertBox( alertStyle: NSAlert.Style.critical, message: "This app uses iCloud to store your connection settings. Please sign in to iCloud in System Preferences." )

File: SSH TunnelBuilder/Models/ViewController.swift (hunk 1 above)
Comment (Snoldinator) [reply SSH TunnelBuilder/Models/ViewController.swift]: Coolio - Added the class and function as suggested

File: SSH TunnelBuilder/Models/ViewController.swift (hunk 2 above)
Comment (Snoldinator) [reply SSH TunnelBuilder/Models/ViewController.swift]: Done. And fun was NOT had by all :P

**Pull request merged**